pkg/deployments/tasks/20230403-timelock-authorizer/input/goerli.ts

```diff
@@ -0,0 +1,619 @@
+    },
+    txHash: '0x79102fe6eb2ddddbd19801b99e1438635d14196806420150115ecf6f4e9ccb22',
+  },
+  {
+    id: '0xdf88e29d8c178bfc181bd582ad0971ba90d2bc1b9766c948675c3e0726cc8ec8',
+    account: '0xc92e8bdf79f0507f65a392b0ab4667716bfe0110',
+    action: {
+      id: '0x1282ab709b2b70070f829c46bc36f76b32ad4989fecb2fcb09a1b3ce00bbfc30',
+    },
+    txHash: '0xc8d38f1cf55ad39528e3b982dffec303553be87f175dfc0aa034dfa990e96eae',
+  },
+  {
+    id: '0xebe65f08d697479935a9a87d9c33a16cee56f5b5965316a08cdce75a0d35fd45',
+    account: '0x00e695aa8000df01b8dc8401b4c34fba5d56bbb2',
+    action: {
+      id: '0x7b8a1d293670124924a0f532213753b89db10bde737249d4540e9a03657d1aff',
+    },
+    txHash: '0x4e1a71cd15c0124265e9c686c8af9fc84c3fcefade60d416f7fccb1b11ca1078',
+  },
+  {
+    id: '0xec70f52d9ded31e22a2d0e86c55c0f8ce59113748b94722e911f8f88e084f5c4',
+    account: '0x7b9b6f094dc2bd1c12024b0d9cc63d6993be1888',
+    action: {
+      id: '0x1282ab709b2b70070f829c46bc36f76b32ad4989fecb2fcb09a1b3ce00bbfc30',
+    },
+    txHash: '0x737deb63336a007f7283437e6c05e7877ad1c65554f2002a7dfa8bed63e3301b',
+  },
+  {
+    id: '0xee46d55032aeffcd16fab472298202be13d40127fde9c74ccc538ea6a41f56e4',
+    account: '0x00e695aa8000df01b8dc8401b4c34fba5d56bbb2',
+    action: {
+      id: '0x0014a06d322ff07fcc02b12f93eb77bb76e28cdee4fc0670b9dec98d24bbfec8',
+    },
+    txHash: '0x4e1a71cd15c0124265e9c686c8af9fc84c3fcefade60d416f7fccb1b11ca1078',
+  },
+  {
+    id: '0xf022e2f2466ec1fc146121bd2255903901aaf7e437e839815c7ce9e89a82323b',
+    account: '0x2fb8ad26fadc971464a17407e9213d1a22befc8a',
+    action: {
+      id: '0xeba777d811cd36c06d540d7ff2ed18ed042fd67bbf7c9afcf88c818c7ee6b498',
+    },
+    txHash: '0x8796f9432b2be267c45d90528847f56812ad9889af5473117aa36257263872e9',
+  },
+  {
+    id: '0xf5592629d54c91c5b2a71b8170e0c40e005ac50361df3b15d00e781a85b9aa96',
+    account: '0x0df18b22fb1dd4c1d4bfbf783a8acf0758979328',
+    action: {
+      id: '0xf0a77343972ffaea6782864c7880d1cee9576736637f7702cb0a922223208118',
+    },
+    txHash: '0x3d2ac5880a7cd6d7ab2076c4400a6e043e5b81049f056492a3330e9c389ce6c5',
+  },
+  {
+    id: '0xf901847778347508dad22263fae0325fef264af513329c3accb7b1f7bad23cbd',
+    account: '0x2fb8ad26fadc971464a17407e9213d1a22befc8a',
+    action: {
+      id: '0x0014a06d322ff07fcc02b12f93eb77bb76e28cdee4fc0670b9dec98d24bbfec8',
+    },
+    txHash: '0x022d905489a2b04d48340f9283ed6fd99994d0467a62cdb2d66272e8640dbe53',
+  },
+  {
+    id: '0xfbcd877a9572441db642e94dde0cad4eb9ae31d74a2426d77333ad8d75c4a52e',
+    account: '0x03f1ab8b19bce21eb06c364aec9e40322572a1e9',
+    action: {
+      id: '0x7b8a1d293670124924a0f532213753b89db10bde737249d4540e9a03657d1aff',
+    },
+    txHash: '0x79102fe6eb2ddddbd19801b99e1438635d14196806420150115ecf6f4e9ccb22',
+  },
+];
+
+export const GrantDelays: DelayData[] = [
+  {
+    actionId: BalancerTokenAdmin.actionId('BalancerTokenAdmin', 'mint(address,uint256)'),
+    newDelay: 30 * DAY,
```

Comment:
Let's have shorter delays for goerli, since there's nothing at risk here. We still want to have delays for testing, but they don't need to be as long. The 30/14/7 day tiers could be reduced to 2/1/0.25 days